susemanager/bin/mgr-setup

```diff
@@ -387,13 +422,18 @@ db-name=$MANAGER_DB_NAME
 db-host=$MANAGER_DB_HOST
 db-port=$MANAGER_DB_PORT
 db-protocol=$MANAGER_DB_PROTOCOL
+externaldb=$EXTERNALDB
+externaldb-admin-user=$EXTERNALDB_ADMIN_USER
+externaldb-admin-password=$EXTERNALDB_ADMIN_PASS
+db-ca-cert=$MANAGER_DB_CA_CERT
+report-db-ca-cert=$REPORT_DB_CA_CERT
```

Comment:
Is it possible / should we support that we have 2 different CAs here?
What does AWS provide? Can you have 2 DBs and say you want to use the same CA (different server certs, but signed from the same CA)?
This would make some things a lot of easier.

Reply:
If you are using RDS, it is the same root cert.  Looking at the existing code  you will see originally there were different certs used in Setup.pm

[1)](https://github.com/uyuni-project/uyuni/blob/master/spacewalk/setup/lib/Spacewalk/Setup.pm)  

`$ENV{PGSSLROOTCERT} = $answers->{'report-db-ca-cert'};`

[2)](https://github.com/uyuni-project/uyuni/blob/master/spacewalk/setup/lib/Spacewalk/Setup.pm#L849)

`$ENV{PGSSLROOTCERT} = $answers->{'db-ca-cert'};`

It seems in Setup.pm there exists some code for doing ssl on the manager DB but I do not think that is fully implemented. as the mgr-setup does not have code for SSL on the manager DB.

You can see in mgr-setup [here](https://github.com/uyuni-project/uyuni/blob/master/susemanager/bin/mgr-setup#L396) only one value is read in

`report-db-ca-cert=$CA_PATH/LOCAL-RHN-ORG-TRUSTED-SSL-CERT`

I was just keeping this in place and adding in the missing variable handling for 

`db-ca-cert`. 

Perhaps this gets addressed more adequately during a refactoring of the whole DB setup handling. I was just trying to add RDS with the least amount of impact/changes.  

Can we resolve this or is further discussion needed?

Reply:
I will have a look tomorrow what happens in the setup code if we get 2 different CAs.